src/components/SwitchUserRoleDialog.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <v-dialog max-width="504" persistent>
```

Comment:
this looks like a very random number
maybe change to 500?

Reply:
Idk exactly why I am doing it this way, but I always choose numbers that are a multiple of 24... 480 + 24 = 504. However, I see what you mean -- one sees that number and asks themselves 'Hey, why 504???'. I can change it of course.

Reply:
I thought about this and I've decided to change it to 500 -- this way it's a clean and not-so-random number, I like it.